ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -128,11 +132,10 @@ class CommandBarButton: UIButton {
         accessibilityLabel = (accessibilityDescription != nil) ? accessibilityDescription : title
         accessibilityHint = item.accessibilityHint
         accessibilityValue = item.accessibilityValue
-        accessibilityIdentifier = item.accessibilityIdentifier
     }
 
     private let isPersistSelection: Bool
-
+/*
```

Comment:
Why are these lines of code commented out?

Reply:
looks like not all the changes went in, sorry I might have missed out on why that is

Reply:
I was moving to fast is why :) Will fix this momentarily!

Reply:
Totally understand! Just making sure :D